src/script/microbit-interfacing/Microbits.ts

```diff
@@ -549,7 +600,8 @@ class Microbits {
     if (!this.isOutputAssigned() || !this.isInputAssigned()) {
       return false;
     }
-    return this.getInput().getDevice().id == this.getOutput().getDevice().id;
+    return false;
+    // return this.getInput().getDevice().id == this.getOutput().getDevice().id;
```

Comment:
Just noticed this, didn't see it before. I would guess `this.getInput().getDevice().id == this.getOutput().getDevice().id` probably returns false, as I assume the current connection flow doesn't assign an output micro:bit, but I haven't checked.

Reply:
The problem is there's nothing on MicrobitConnection that exposes a device id. I'll add something suitable in a mo.

Reply:
a5327547178e2bc290e604651261aefc2ab634dc

Reply:
FWIW I don't think we'll run this code as we'll never have an output micro:bit but at least now it compiles and the intent is preserved.